src/RulesTest/PropertyConditions/NameTest.cs

```diff
@@ -22,20 +22,20 @@ public NameTest()
             RequiredTypes = new List<int>
             {
                 Calendar, CheckBox, ComboBox,
-               DataGrid, DataItem, Document,
```

Comment:
With the exception of the marked block at the end of this file, these change are all correcting indentation issues